updater/README.md

```diff
@@ -11,30 +11,37 @@ GitHub network, and so is not generally accessible.
 
 ## Setup
 
-You will need to provide the build a Personal Access Token to access the GitHub Package Registry to retrieve
-dependency containers.
+To work on the Updater, you will need to start a Docker dev shell. The bundler
+shell will suffice
 
-[Create a token](https://github.com/settings/tokens/new) with the `packages:read` scope and set it in your environment
-as `GPR_TOKEN`
-
-Run the setup script:
-
-```
-script/setup
+```zsh
+➜ bin/docker-dev-shell bundler
+[dependabot-core-dev] ~/dependabot-core $ cd updater/
+[dependabot-core-dev] ~/dependabot-core/updater $ bundle
 ```
 
 ## Tests
 
-We run [rspec](https://rspec.info/) tests inside a Docker container for this project:
+We run [rspec](https://rspec.info/) tests in the docker dev shell:
 
-```
-script/test
+```zsh
+[dependabot-core-dev] ~/dependabot-core/updater $ bundle exec rspec
 ```
 
 You can run an individual test file like so:
 
+```zsh
+[dependabot-core-dev] ~/dependabot-core/updater $ bundle exec rspec spec/dependabot/integration_spec.rb
 ```
-script/test spec/dependabot/integration_spec.rb
+
+A small number of tests hit the GitHub API, so you will need to set the envvar
+`DEPENDABOT_TEST_ACCESS_TOKEN` with a Personal Access Token with the full `repo`
+scope.
+
+```zsh
+export DEPENDABOT_TEST_ACCESS_TOKEN=ghp_xxx
```

Comment:
Since we're suggesting using a PAT, it makes sense to me to suggest keeping secrets out of shell history here. What do you think?

```suggestion
# keep secrets from being stored in shell history by prefixing with a space
export HISTCONTROL=ignorespace
export  DEPENDABOT_TEST_ACCESS_TOKEN=ghp_xxx
```

Reply:
Yeah, that's a good idea 👍🏻